docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,256 @@
+    [Nerves section] on [Elixir Forum] or the #nerves channel on the [Elixir
+    Slack].
+
+3.  Questions on [Elixir Forum] (ideally in the [Nerves section])
+
+    If questions and answers are captured in a forum like this, it makes them
+    much easier for people to find later via search engine, as opposed to
+    searching through a real-time conversation stream like Slack.
+
+4.  Blog about your project
+
+    We love to see how people are using Nerves for fun and and for work. If you
+    blog about a Nerves-based project, please consider posting a link somewhere
+    that the community will notice it, like [Elixir Forum], [ElixirStatus], or
+    the #nerves channel on the [Elixir Slack].
+
+5.  Chat with us on Slack
+
+    We have a #nerves channel on the [Elixir Slack]. We'd be happy to have you
+    drop in and let us know if you have questions or want to talk about a
+    project you're working on, or thinking about working on.
+
+[our website]: http://www.nerves-project.org
+[Getting Started Guide]: https://hexdocs.pm/nerves/getting-started.html
+[Nerves system]: https://hexdocs.pm/nerves/systems.html
+[documentation source]: https://github.com/nerves-project/nerves/tree/master/docs
+[website source]: https://github.com/nerves-project/nerves-project.github.com
+[Triage process]: #issue-and-pull-request-triage
+[repositories]: https://github.com/nerves-project
+[nerves repository]: https://github.com/nerves-project/nerves
+[Elixir Forum]: https://elixirforum.com
+[Nerves section]: https://elixirforum.com/c/dedicated-sections/nerves
+[ElixirStatus]: http://elixirstatus.com/
+[security@nerves-project.org]: mailto:security@nerves-project.org
+
+## Answer questions on the Elixir Forum and Slack channel
+
+It takes a lot of time and energy to understand the context of a problem and
+determine an appropriate solution. If you see a question posted on the [Elixir
+Forum], you can help by:
+
+* Making sure it gets marked as belonging to the [Nerves section]
+* Asking clarifying questions to ensure that others will understand the context
+* Offering your own advice or solutions if you can
+* Pinging the core team in the #nerves-dev channel on Slack if it's stale
+
+If you see a question posted in the #nerves channel on Slack, you can help by:
+
+* Asking clarifying questions to ensure that others will understand the context
+* Politely reminding the person that the answer will be more easily found later
+by others if it's posted to the [Elixir Forum] instead
+* Helping to get it posted to the Forum or converted to a GitHub Issue to
+capture or summarize the discussion in Slack about it
+
+## Become a Backer or Sponsor through OpenCollective
+
+The Nerves project has set up an [OpenCollective site] that allows individuals
+and companies to make one-time or recurring financial contributions to cover the
+cost of maintaining the project (e.g. paying for hosted services like AWS and
```

Comment:
Could you delete the (e.g. ...) part? Not that it's incorrect or anything, but I hoping to keep the place for what funds were used on in one place on the OpenCollective site, so that it's easy to update.